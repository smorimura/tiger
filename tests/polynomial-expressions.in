result := 0;
result = 1 + 2 - 3 * 4 / 5;
sayInt(result); sayNewline();
result = 1 + 2 - 3 / 4 * 5;
sayInt(result); sayNewline();
result = 1 - 2 + 3 / 4 * 5;
sayInt(result); sayNewline();
result = 1 - 2 + 3 * 4 / 5;
sayInt(result); sayNewline();
result = 1 + 2 * 3 - 4 / 5;
sayInt(result); sayNewline();
result = 1 + 2 / 3 - 4 * 5;
sayInt(result); sayNewline();
result = 1 - 2 / 3 + 4 * 5;
sayInt(result); sayNewline();
result = 1 - 2 * 3 + 4 / 5;
sayInt(result); sayNewline();
result = 1 + 2 * 3 / 4 - 5;
sayInt(result); sayNewline();
result = 1 - 2 * 3 / 4 + 5;
sayInt(result); sayNewline();
result = 1 - 2 / 3 * 4 + 5;
sayInt(result); sayNewline();
result = 1 + 2 / 3 * 4 - 5;
sayInt(result); sayNewline();
result = 1 * 2 + 3 - 4 / 5;
sayInt(result); sayNewline();
result = 1 * 2 - 3 + 4 / 5;
sayInt(result); sayNewline();
result = 1 / 2 - 3 + 4 * 5;
sayInt(result); sayNewline();
result = 1 / 2 + 3 - 4 * 5;
sayInt(result); sayNewline();
result = 1 * 2 + 3 / 4 - 5;
sayInt(result); sayNewline();
result = 1 / 2 + 3 * 4 - 5;
sayInt(result); sayNewline();
result = 1 / 2 - 3 * 4 + 5;
sayInt(result); sayNewline();
result = 1 * 2 - 3 / 4 + 5;
sayInt(result); sayNewline();
result = 1 * 2 / 3 + 4 - 5;
sayInt(result); sayNewline();
result = 1 * 2 / 3 - 4 + 5;
sayInt(result); sayNewline();
result = 1 / 2 * 3 - 4 + 5;
sayInt(result); sayNewline();
result = 1 / 2 * 3 + 4 - 5;
sayInt(result); sayNewline();
