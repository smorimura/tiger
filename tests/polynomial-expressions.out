0.6
-0.75
2.75
1.4
6.2
-18.333
20.333
-4.2
-2.5
4.5
3.333
-1.333
4.4
-0.2
17.5
-16.5
-2.25
7.5
-6.5
6.25
-0.333
1.666
2.5
0.5